python-2.7.15/python-2.7.15.json

```diff
@@ -0,0 +1,46 @@
+{
+    "name": "python-2.7",
+    "sources": [
+        {
+            "type": "archive",
+            "url": "https://www.python.org/ftp/python/2.7.15/Python-2.7.15.tgz",
+            "sha256": "18617d1f15a380a919d517630a9cd85ce17ea602f9bbdc58ddc672df4b0239db"
+        }
+    ],
+    "buildsystem": "autotools",
+    "confgi-opts": [
+        "--enable-shared",
+        "--with-ensurepip=yes",
+        "--with-system-expat",
+        "--with-system-ffi",
+        "--enable-loadable-sqlite-extensions",
+        "--with-dbmliborder=gdbm",
+        "--enable-unicode=ucs4"
+    ],
+    "cleanup": [
+        "/bin/2to3*",
+        "/bin/easy_install*",
+        "/bin/idle*",
+        "/bin/python*-config",
+        "/bin/pyvenv*",
+        "/bin/pydoc*",
+        "/include",
+        "/share",
+        "/lib/pkgconfig",
+        "/lib/python*/config-*",
+
+        /* Test scripts */
+        "/lib/python*/test",
+        "/lib/python*/*/test",
+        "/lib/python*/*/tests",
+        "/lib/python*/lib-tk/test",
+        "/lib/python*/lib-dynload/_*_test.*.so",
+        "/lib/python*/lib-dynload/_test*.*.so",
+
+        /* Unused modules */
+        "/lib/python*/idlelib",
+        "/lib/python*/tkinter",
+        "/lib/python*/turtle*",
```

Comment:
I can see that at least Tkinter.pyo, Tkinter.pyc and Tkinter.py are still there, same about turtle.*.

Reply:
Weird, I see none of those, I added a `tkinter*` just to make sure though.